src/Data/Monoid/LCM.hs

```diff
@@ -7,19 +7,23 @@
 -- least monoid from which either argument can be subtracted with the '</>'
 -- operation.
 --
--- The 'LCMMonoid' class is for Abelian, /i.e./, 'Commutative' monoids.
```

Comment:
Did you mean to remove this line of comment?

Reply:
Hi @blamario 

Yes, that was actually intentional.

It's a bit awkward to see from the diff, but [this commit](https://github.com/blamario/monoid-subclasses/pull/48/commits/1b5b24482183b607c770d4eff13a1c70601c104f) also adjusts the last line of the module documentation to state that all classes in the `LCM` module are for `Commutative` (Abelian) monoids:
https://github.com/blamario/monoid-subclasses/blob/1b5b24482183b607c770d4eff13a1c70601c104f/src/Data/Monoid/LCM.hs#L13

When rendered with Haddock, it looks like this:
![Screenshot from 2023-03-31 12-43-44](https://user-images.githubusercontent.com/206319/229024698-e23b3269-d2e1-4dac-8d77-327fc72d382b.png)

Does the wording seem sensible to you? (Happy to make any adjustments, if you have a different preferred wording!)

Reply:
It's all clear, I should have clued in myself.